app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -52,9 +52,8 @@ def index
         version = ContentViewVersion.in_environment(params[:environment_id]).where(:content_view_id => params[:content_view_id])
         repositories = repositories.where(:content_view_version_id => version)
       elsif params[:content_view_id]
-        repositories = repositories
-                         .joins(:content_view_repositories)
-                         .where("#{ContentViewRepository.table_name}.content_view_id" => params[:content_view_id])
+        repositories = repositories.joins(:content_view_repositories).
+            where("#{ContentViewRepository.table_name}.content_view_id" => params[:content_view_id])
```

Comment:
This method has too many lines - 1 and I didn't think refactoring the method would be appropriate for this bug fix so I'm making room for my two lines.